deployments/helm-chart/templates/rbac.yaml

```diff
@@ -62,7 +62,7 @@ rules:
   - patch
   - list
 - apiGroups:
-  - extensions
+  - networking.k8s.io
```

Comment:
the changes to the rbac in Helm must be the same as the ones in deployments/rbac/rbac.yaml